src/govuk/components/accordion/accordion.test.js

```diff
@@ -120,6 +125,8 @@ describe('/components/accordion', () => {
 
       describe('"+/-" icon', () => {
         it('should display "+/-" in the section headings', async () => {
+          console.log(await page.evaluate(() => JSON.stringify(window.sessionStorage)))
```

Comment:
Has this been added intentionally?

Reply:
Ah, no, that was me experimenting, thanks for catching that.

Reply:
I've removed the offending line 😁